source/_patterns/basics/forms/static-controls.hbs

```diff
@@ -10,7 +10,8 @@
       </div>
     </div>
 
```

Comment:
Maybe for future discussion, I'm curious if there would be any value in having a handlebar for the static control form group.